### PR TITLE
libvirt,docs: Make console workaround a bit easier

### PR DESCRIPTION
**Pull request by zeenix**
Instead of telling what the content of a manifest should be like, provide a command to modify it as needed.

### CODE REVIEW

**Comment by sgreene570** [comment]
I'm confused, isn't the grep command just verifying that the manifests were populated correctly? Is this sed command doing the proper replacement, even if the file is OK as is?

File: docs/dev/libvirt/README.md [hunk 1]
```diff
@@ -299,8 +299,8 @@ $ openshift-install --dir $INSTALL_DIR create manifests
 
 - Domain entry for cluster-ingress-02-config.yml file should be following (here domain is what your created initially)
```
Comment (patrickdillon) [inline docs/dev/libvirt/README.md]: This line no longer makes sense with your change. Can you update it to reflect the changes you have made?

Comment (zeenix) [reply docs/dev/libvirt/README.md]: sorry for the late response, I was on vacation.

Comment (zeenix) [reply docs/dev/libvirt/README.md]: > I'm confused, isn't the grep command just verifying that the manifests were populated correctly? Is this sed command doing the proper replacement, even if the file is OK as is?

I just forgot to update the line before the `sed` command as @patrickdillon pointed out. I'll fix that..

**Comment by patrickdillon** [comment]
scaleup is broken, waiting for a fix in CRIO RPMS
/skip

**Comment by zeenix** [comment]
@patrickdillon Thanks for the reviews. :)

**Pull request merged**